pkg/monitor/cluster/clusterversions_test.go

```diff
@@ -12,17 +12,20 @@ import (
 	"github.com/openshift/client-go/config/clientset/versioned/fake"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
+	"github.com/Azure/ARO-RP/pkg/api"
 	mock_metrics "github.com/Azure/ARO-RP/pkg/util/mocks/metrics"
 )
 
 func TestEmitClusterVersion(t *testing.T) {
 	ctx := context.Background()
 
 	for _, tt := range []struct {
-		name               string
-		cv                 *configv1.ClusterVersion
-		wantActualVersion  string
-		wantDesiredVersion string
+		name                        string
+		cv                          *configv1.ClusterVersion
```

Comment:
is this field actually used?

Reply:
yeah, it's fed into https://github.com/Azure/ARO-RP/blob/dfb81df500850edc7b1b924433608a8c94244ff4/pkg/monitor/cluster/clusterversions.go#L14

Reply:
I think you're mistaken ;-)
You add the field to the struct and initialise it at line 88, but it's not actually read anywhere?

Reply:
Ugh, ignore me, I'm wrong, I got confused by the indentation change.  Sorry for the noise.

Reply:
In the GitHub diff UI, you can enable this, which reduces whitespace noise (like this) from gofmt changes:

![image](https://user-images.githubusercontent.com/3307100/86299786-7716c380-bc44-11ea-8a5b-6d5e8ad74650.png)